rts/lra/README.adoc

```diff
@@ -1,5 +1,7 @@
 ## Build the demo
 
+Make sure you export the WORKSPACE variable to the absolute path to the quickstart folder, e.g. 'export WORKSPACE=/user/myuser/quickstart/'.
```

Comment:
Is it possible to get the script to fail when this is not set? Trying it here it basically seemed to hang when I did not have it set so it might be better if the script (run.sh?) checks if the variable is set and if not results in a exit -1 or something like that?

Reply:
Thanks Tom for your suggestion, I have updated the PR.

Reply:
Thank you